src/Clients/ClientHelper.php

```diff
@@ -5,6 +5,7 @@
 use Resiliency\Contracts\Client;
 use Resiliency\Contracts\Place;
 use Resiliency\Contracts\Service;
+use Psr\Http\Message\ResponseInterface;
 
 abstract class ClientHelper implements Client
```

Comment:
we must implement ClientInterface from PSR-18.

Reply:
Do you want to remove the current Client interface?
Could it "extend" it instead maybe? (just thinking out loud)

Reply:
Ideally, I want to remove it: less code to maintain => less code that can bug or break.

Honest, I don't see value keeping my own interface if I don't needs hacks to manage the HTTP method or arguments.

Also, the former implementation is unable to deal with streamable responses (afaik): it's probably a super tricky rare use case but... I'm close to support this use case too